rudder_airflow_provider/hooks/rudderstack.py

```diff
@@ -47,11 +47,9 @@ def poll_for_status(self):
         while True:
             self.method = 'GET'
             resp = self.run(endpoint=status_endpoint, headers=headers).json()
-            job_status = resp['status']
-            logging.info('sync status for sourceId: %s, status: %s',
-                         self.source_id, job_status)
-
-            if job_status == STATUS_FINISHED:
+            finish_time = resp.get(FINISHED_AT) or None
```

Comment:
can we make this change in the API instead of provider?

Reply:
and `finished_at` will be '0001-01-01 00:00:00'

Reply:
Fixed.